<?xml version="1.0" encoding="UTF-8" ?>
<testsuites errors="1"
            failures="0"
            tests="1">

  <testsuite name="artsytestcases.testcase2"
    errors="1" failures="0" hostname="" id="" package="artsytestcases" skipped="0"
    tests="1" time="0.009000" timestamp="Mon, 11 Dec 2017 22:28:08 GMT">
  
    <testcase name="Go to the Search box and search for specific term" classname="artsytestcases.testcase2" time="0.009000" assertions="0">
    </testcase>
  

  
    <system-err>
        Error while running [Artsytestcases / Testcase2 / Go to the Search box and search for specific term]:

  ReferenceError: assert is not defined
    at Object.Go to the Search box and search for specific term (/Users/Ronak/Desktop/learnautomation/tests/artsytestcases/testcase2.js:10:9)
    at Module.call (/Users/Ronak/Desktop/learnautomation/node_modules/nightwatch/lib/runner/module.js:62:34)
    at /Users/Ronak/Desktop/learnautomation/node_modules/nightwatch/lib/runner/testcase.js:70:29
    at _fulfilled (/Users/Ronak/Desktop/learnautomation/node_modules/q/q.js:834:54)
    at self.promiseDispatch.done (/Users/Ronak/Desktop/learnautomation/node_modules/q/q.js:863:30)
    at Promise.promise.promiseDispatch (/Users/Ronak/Desktop/learnautomation/node_modules/q/q.js:796:13)
    at /Users/Ronak/Desktop/learnautomation/node_modules/q/q.js:556:49
    at runSingle (/Users/Ronak/Desktop/learnautomation/node_modules/q/q.js:137:13)
    at flush (/Users/Ronak/Desktop/learnautomation/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    </system-err>
  

  
  </testsuite>
</testsuites>
